src/ConnectedMode/UI/ServerSelection/ServerSelectionViewModel.cs

```diff
@@ -72,11 +71,11 @@ public string SonarQubeUrl
         private bool IsSonarQubeUrlProvided => !string.IsNullOrWhiteSpace(SonarQubeUrl);
         public bool ShowSecurityWarning => Uri.TryCreate(SonarQubeUrl, UriKind.Absolute, out Uri uriResult) && uriResult.Scheme != Uri.UriSchemeHttps;
 
-        public Connection CreateConnection()
+        public ConnectionInfo CreateConnection()
```

Comment:
Now that we have a "Connection" and "ConnectionInfo" types, maybe we should rename the method to CreateConnectionInfo.

Reply:
Fixed